target/linux/ramips/image/mt7621.mk

```diff
@@ -1876,6 +1876,18 @@ define Device/zbtlink_zbt-wg1602-16m
 endef
 TARGET_DEVICES += zbtlink_zbt-wg1602-16m
 
+define Device/zbtlink_zbt-wg1608-16m
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := Zbtlink
+  DEVICE_MODEL := ZBT-WG1608
+  DEVICE_VARIANT := 16M
+  DEVICE_PACKAGES := kmod-sdhci-mt7620 kmod-mt7603 kmod-mt7615e \
+	kmod-mt7663-firmware-ap kmod-usb3 kmod-usb-ledtrig-usbport
```

Comment:
Do you need `kmod-mt7615e kmod-mt7663-firmware-ap kmod-usb-ledtrig-usbport`?

Reply:
```kmod-mt7615e kmod-mt7663-firmware-ap``` is required for the device's 5Ghz WiFi to work. 
```
[   18.938986] mt7615e 0000:02:00.0: mediatek/mt7663pr2h.bin not found, switching to mediatek/mt7663pr2h_rebb.bin
[   19.319900] mt7615e 0000:02:00.0: HW/SW Version: 0x65322d31, Build Time: 2009041715da1a1
[   19.319900]
[   19.759847] mt7615e 0000:02:00.0: N9 Firmware Version: 7663mp1827, Build Time: 20200904171623
[   19.768426] mt7615e 0000:02:00.0: Region number: 0x3
[   19.773414] mt7615e 0000:02:00.0: Parsing tailer Region: 0
[   19.781967] mt7615e 0000:02:00.0: Region 0, override_addr = 0x00112c00
[   19.788532] mt7615e 0000:02:00.0: Parsing tailer Region: 1
[   19.794860] mt7615e 0000:02:00.0: Parsing tailer Region: 2
[   19.800678] mt7615e 0000:02:00.0: override_addr = 0x00112c00, option = 3
```

Reply:
```kmod-usb-ledtrig-usbport``` is needed for the wwan LED, but the device has many wwan card variants, so I couldn't make default setting.

Reply:
> `kmod-mt7615e kmod-mt7663-firmware-ap` is required for the device's 5Ghz WiFi to work.

Oh, so mt7613 uses kmod-mt7663-firmware-ap... But still it won't require kmod-mt7615e, does it?

> `kmod-usb-ledtrig-usbport` is needed for the wwan LED, but the device has many wwan card variants, so I couldn't make default setting.

I see. But I guess all you need is just `linux,default-trigger = "usbport"` and `trigger-sources = <&ehci_port2>` in the wwan node, no?

Reply:
> I see. But I guess all you need is just `linux,default-trigger = "usbport"` and `trigger-sources = <&ehci_port2>` in the wwan node, no?

I added it

Reply:
You said there's a USB 3.0 port so I guessed the wwan card uses only 2.0. If you use `&xhci_ehci_port1` then the LED will be triggered by the USB 3.0 port, isn't it?

Reply:
wwan card also uses usb3.

Reply:
Hmm that's unexpected, then both wwan card and USB port cannot be used simultaneously?

Reply:
I checked. this board has built-in usb hub.
```
/:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=xhci-mtk/1p, 5000M
    ID 1d6b:0003
    |__ Port 1: Dev 2, If 0, Class=, Driver=hub/4p, 5000M
        ID 05e3:0626
        |__ Port 2: Dev 3, If 0, Class=, Driver=, 5000M
            ID 2c7c:0800
        |__ Port 2: Dev 3, If 1, Class=, Driver=, 5000M
            ID 2c7c:0800
        |__ Port 2: Dev 3, If 2, Class=, Driver=, 5000M
            ID 2c7c:0800
        |__ Port 2: Dev 3, If 3, Class=, Driver=, 5000M
            ID 2c7c:0800
        |__ Port 2: Dev 3, If 4, Class=, Driver=qmi_wwan, 5000M
            ID 2c7c:0800
        |__ Port 4: Dev 4, If 0, Class=, Driver=, 5000M
            ID 090c:1000
/:  Bus 01.Port 1: Dev 1, Class=root_hub, Driver=xhci-mtk/2p, 480M
    ID 1d6b:0002
    |__ Port 1: Dev 2, If 0, Class=, Driver=hub/4p, 480M
        ID 05e3:0610
```

Reply:
Thanks, that explains.